.circleci/config.yml

```diff
@@ -24,6 +24,23 @@ jobs:
         paths:
         - venv/
         key: python-packages-v1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "constraints.txt" }}
+  verify-generated-sql:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Verify that all of generated SQL is committed
+          command: |
+            sudo apt-get update
+            python3.7 -m venv venv/
+            venv/bin/pip install --upgrade -r requirements.txt
```

Comment:
Looking at the build output for this job in CI, it's pulling in packages that it doesn't need. Since generate_sql only needs the standard library, we could avoid calling pip at all and the job would run faster.

Reply:
`generate_sql` calls `parse_udf` which requires some packages like `sqlparse`

Reply:
I could check which packages are actually required and only install those if that's better?

Reply:
No, I'm just wrong on this one 🤦‍♂ 

I think you're doing exactly the right thing to install via requirements.txt so we don't have extra logic to maintain here for this particular test.